sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/src/Vendored/HttpClientMetricInstrumentationOptions.cs

```diff
@@ -0,0 +1,43 @@
+#pragma warning disable SA1636
+// <copyright file="HttpClientMetricInstrumentationOptions.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System.Diagnostics;
+using Microsoft.Extensions.Configuration;
+using static OpenTelemetry.Internal.HttpSemanticConventionHelper;
+
+namespace OpenTelemetry.Instrumentation.Http;
+
+internal sealed class HttpClientMetricInstrumentationOptions
```

Comment:
this not being public prevents any customization :(

Reply:
Sure that is plausible. Just very inconvenient for end-users.

Reply:
it will be convenient once instrumentation becomes stable.  
In the meantime, I guess it will affect a small subset of users, allowing the rest of them (who use defaults) to never worry about version compat, updates, etc.